components/admin_console/group_settings/group_details/group_details.jsx

```diff
@@ -137,10 +137,17 @@ export default class GroupDetails extends React.PureComponent {
         const {group} = this.props;
         const originalAllowReference = group.allow_reference;
         const saveNeeded = true;
+        let {groupMentionName} = this.state;
+
+        if (!originalAllowReference && allowReference &&
+            groupMentionName.length === 26 && new RegExp('^[a-z0-9]+$').test(groupMentionName)) {
```

Comment:
can we define `26` as a constant something like `GROUP_ID_LENGTH` to avoid any confusion as to why we're comparing with an arbitrary number

Reply:
Yes, thank you. Meant to do that.